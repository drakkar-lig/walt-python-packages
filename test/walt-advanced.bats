
@test "walt advanced sql" {
    result="$(echo "\dt" | walt advanced sql | grep -c logstreams)"
    [ "$result" -eq 1 ]
}
