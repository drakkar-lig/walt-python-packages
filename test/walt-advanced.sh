
define_test "walt advanced sql" as {
    result="$(echo "\dt" | walt advanced sql | grep -c logstreams)"
    [ $result -eq 1 ]
}

define_test "walt advanced update-hub-meta" as {
    walt advanced update-hub-meta
}
