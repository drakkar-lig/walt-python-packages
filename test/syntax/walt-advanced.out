
WalT platform control tool
-> advanced sub-commands

Usage:
    walt advanced SUBCOMMAND [args...]

Help about a given subcommand:
    walt advanced SUBCOMMAND --help

Help about WalT in general:
    walt help show

Sub-commands:
    fix-image-owner     fix the owner of images
    rescan-hub-account  alias to 'update-hub-meta' subcommand
    sql                 Start a remote SQL prompt on the WalT server database
    update-hub-meta     update hub metadata (docker images pushed without walt)
